### ISSUE TITLE
Broken on Fedora 37

### ISSUE THREAD

**Issue by azzid** [flathub/im.riot.Riot#329]
Just installed Fedora 37. First application to install: Element.

Enabled flathub:
```
$ flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
```

Downloaded and installed: https://dl.flathub.org/repo/appstream/im.riot.Riot.flatpakref

Application keep crashing after a few seconds-minutes.

Error as reported by abrt:

![image](https://user-images.githubusercontent.com/8883008/215119657-4bd41052-326a-4449-ab0d-96fc2d1e6aa1.png)

**Comment by espidev** [comment]
I seem to get this issue frequently as well.

**Comment by martijk** [comment]
This might be related to the message search functionality. I remember that disabling that option worked for me some weeks back when I had the same issue (see Security & Privacy in Settings). I've since reinstalled my OS (for unrelated reasons) and started with a fresh `~/.var/app/im.Riot.riot` directory, and I haven't experienced any issues since. But then again, the search functionality isn't working either, so that could still be the cause of the frequent crashes. This is the error why the search functionality is currently broken on my system:

```
Error opening the database: DatabaseUnlockError("Invalid passphrase")
```

I'm not going to fix this error now and risk having the app crash again because I use this application for my job.

**Comment by SISheogorath** [comment]
This should be resolved by #357. If the issue persists, feel free to reopen :)
